frontend/src/component/feature/FeatureToggleList/FeatureToggleFilters/FeatureToggleFilters.tsx

```diff
@@ -49,6 +52,7 @@ export const FeatureToggleFilters: VFC<IFeatureToggleFiltersProps> = ({
 }) => {
     const { projects } = useProjects();
     const { segments } = useSegments();
+    const { tags } = useTags('.');
```

Comment:
This will call /api/admin/tags/,  which returns all tags.

Reply:
I'd hide it at the source and expose clean API for clients

Reply:
Added new hook. useAllTags